packages/form/test/lion-form.test.js

```diff
@@ -7,7 +7,7 @@ import '@lion/fieldset/lion-fieldset.js';
 import '../lion-form.js';
 
 describe('<lion-form>', () => {
-  it.skip('has a custom reset that gets triggered by native reset', async () => {
+  it('has a custom reset that gets triggered by native reset', async () => {
     const withDefaults = await fixture(html`
       <lion-form
```

Comment:
the submit and reset events are a public api and should be tested with something like this
```js
   it('dispatches [submit, reset] events', async () => {
    const el = await fixture(html`
      <${tag}></${tag}>
    `);
    setTimeout(() => el.submit());
    const submitEv = await oneEvent(el, 'submit');
    expect(submitEv).to.be.instanceOf(Event);
    expect(submitEv.type).to.equal('submit');
    expect(submitEv.target).to.equal(el);
    expect(submitEv.bubbles).to.be.true;
    expect(submitEv.composed).to.be.false;

    setTimeout(() => el.reset());
    const resetEv = await oneEvent(el, 'reset');
    expect(resetEv).to.be.instanceOf(Event);
    expect(submitEv.type).to.equal('reset');
    expect(resetEv.target).to.equal(el);
    expect(resetEv.bubbles).to.be.true;
    expect(resetEv.composed).to.be.false;
  });
```